.goreleaser.yml

```diff
@@ -2,8 +2,6 @@ project_name: regen-ledger
 
 env:
   - GO111MODULE=on
-  - CGO_ENABLED=1
-  - LD_LIBRARY_PATH=/home/runner/work/osxcross/target/binaries/lib
```

Comment:
`CGO_ENABLED` is not applicable to arm builds so setting within specific builds.

Reply:
`LD_LIBRARY_PATH` is only applicable to darwin builds.